### DISCUSSION TITLE
BCU-console use

### DISCUSSION THREAD

**Discussion by ecrofeht** [Klocman/Bulk-Crap-Uninstaller#629] [Q&A]
I have been trying to get this to work and am very interested in using this across 1000+ systems. I am using silverlight as a test for removal but cannot seem to get bcu-console to actually remove it. Even if I use the /V it prompts uac and then nothing happens. I have the bcul file and it works in gui. Basically I have a list of programs that need to be removed if installed and this would save me thousands of manhours if I can make it pull a list and remove anything on that list without any user interaction all from a script.

**Comment by Klocman** [comment]
That's strange, does it fail to detect the application or to uninstall it? Uninstalling should be the same as in the GUI, same with logic that detects the applications. I'd guess it's something to do with commandline args? Maybe you have simulation enabled?
